Transportation/Vehicle/Vehicle/schema.json

```diff
@@ -85,13 +85,22 @@
           "type": "string"
         },
         "dateVehicleFirstRegistered": {
-          "type": "date"
+          "type": "array",
+          "enum": [
+            "date"
+          ]
         },
         "dateFirstUsed": {
-          "type": "date"
+          "type": "array",
+          "enum": [
+            "date"
+          ]
         },
         "purchaseDate": {
-          "type": "date-time"
+          "type": "array",
```

Comment:
this should not be an array. it should be a string with format date-time.

"type": "string",
"format": "date-time"